disnake/client.py

```diff
@@ -271,22 +271,37 @@ def __init__(
         *,
         asyncio_debug: bool = False,
         loop: Optional[asyncio.AbstractEventLoop] = None,
-        **options: Any,
+        shard_id: Optional[int] = None,
+        shard_count: Optional[int] = None,
+        enable_debug_events: bool = False,
+        connector: Optional[aiohttp.BaseConnector] = None,
+        proxy: Optional[str] = None,
+        proxy_auth: Optional[aiohttp.BasicAuth] = None,
+        assume_unsync_clock: bool = True,
+        max_messages: Optional[int] = 1000,
+        application_id: Optional[int] = None,
+        heartbeat_timeout: float = 60.0,
+        guild_ready_timeout: float = 2.0,
+        allowed_mentions: Optional[AllowedMentions] = None,
+        activity: Optional[BaseActivity] = None,
+        status: Optional[Union[Status, str]] = None,
+        intents: Intents = Intents.default(),
```

Comment:
Not sure about having a mutable default here; it's probably fine given the reference isn't exposed in a public attribute anywhere, but defaulting to `None` or `...` is still preferable.

Reply:
Set the default to `None`, as this was already handled by `ConnectionState` to default to `Intents.default()`.